test/factories/accounts/user_factory.ex

```diff
@@ -4,11 +4,15 @@ defmodule GoogleCrawler.Accounts.UserFactory do
       defp user_email, do: "user#{System.unique_integer()}@example.com"
       defp user_password, do: "hello world!"
 
-      def user_factory do
-        %GoogleCrawler.Accounts.User{
-          email: user_email(),
-          hashed_password: Bcrypt.hash_pwd_salt(user_password())
+      def user_factory(attrs) do
+        user = %GoogleCrawler.Accounts.User{
+          email: attrs[:email] || user_email(),
+          hashed_password: Bcrypt.hash_pwd_salt(attrs[:password] || user_password())
         }
+
+        user
+        |> merge_attributes(Map.delete(attrs, :password))
+        |> evaluate_lazy_attributes()
```

Comment:
Curious if something like this works? 🤔 

```suggestion
       password = attrs[:password] || default_user_password()
       
        %GoogleCrawler.Accounts.User{
          email: attrs[:email] || user_email(),
          hashed_password: Bcrypt.hash_pwd_salt(password)
        }
```